cluster/examples/kubernetes/ceph/csi/template/cephfs/csi-cephfsplugin-provisioner-dep.yaml

```diff
@@ -12,6 +12,7 @@ spec:
     metadata:
       labels:
         app: csi-cephfsplugin-provisioner
+        driver: ceph-csi
```

Comment:
Is this part of the fix? If not please make it a separate commit.

Reply:
Can you please update the commit message. why we need this label?

Reply:
@rohan47 It would be simple just to add the csi labels to the list in `getPodsOnNode()` instead of adding this new label. I know we discussed this label, but it seems simple enough without it.

Reply:
If this label addition is kept, make sure to test upgrades. I think it is the `selector:matchLabels` that can't be change with a k8s deployment update call, but I could be mistaken.

Reply:
added the csi labels to the list in `getPodsOnNode()`